app/frontend/src/locales/es/translation.json

```diff
@@ -81,6 +81,10 @@
         "minimumSearchScore": "Puntaje mínimo de búsqueda",
         "minimumRerankerScore": "Puntaje mínimo de re-clasificación",
         "retrieveCount": "Obtén éste número resultados de búsqueda:",
+        "includeCategory": "Incluir categoría",
+        "includeCategoryOptions": {
+            "all": "Todos"
```

Comment:
```suggestion
            "all": "Todas"
```

Reply:
https://github.com/Azure-Samples/azure-search-openai-demo/pull/2021#discussion_r1806787049 is accepted .